
<TranslatedText
	xmlns:cws="http://schemas.cordys.com/cws/1.0"
	type="com.cordys.translation.TranslatedText"
	RuntimeDocumentID="98FA9B1A-7A8B-A1EA-B2C7-5FFAE2C44378">
	<DocumentID>98FA9B1A-7A8B-A1EA-B2C7-5FFAE2C40378</DocumentID>
	<Name>98FA9B1A-7A8B-A1EA-B2C7-5FFAE2C40378</Name>
	<Description>Catch Exception</Description>
	<CreatedBy>sysadmin</CreatedBy>
	<CreationDate>2020-07-21T06:53:56.878</CreationDate>
	<Notes/>
	<Text>Catch Exception</Text>
	<Language>
		<uri
			id="484D7EE5-DBB0-A1E8-ABD4-86C0D9D1A4F0"
		/>
	</Language>
	<TextIdentifier>
		<uri
			id="98FA9B1A-7A8B-A1EA-B2C7-5FFAE2C38378"
		/>
	</TextIdentifier>
</TranslatedText>
